Automattic/jetpack
# VideoPress: fix bug when site contains local videos

This PR fixes a bug that happens when the site has local videos.
In short, the app tries to pull a VideoPress video even when rendering the local videos library.
The suggested solution is adding a LocalVideoList component that renders a more straightforward list just for local videos.

The implementation deserves more love, but it's okay since it fixes the bug. Let's keep polishing in follow-up PRs.


#### Changes proposed in this Pull Request:

* VideoPress: fix bug when site contains local videos


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashboard
* ensure the site has VideoPress and local videos
* Before these changes, confirm the app get broken in the initial rendering
* After these changes, the app works as usual

before | after
------|------
<img width="1728" alt="Screen Shot 2022-10-13 at 21 21 37" src="https://user-images.githubusercontent.com/77539/195735020-a43b80f9-894e-49b8-ba2e-123ece97e005.png"> | <img width="1728" alt="image" src="https://user-images.githubusercontent.com/77539/195734865-dad9ee83-9c7e-4efd-9ee1-6ed89cb9e326.png">
